src/JustSaying/AwsTools/QueueCreation/SqsReadConfiguration.cs

```diff
@@ -20,6 +22,7 @@ public SqsReadConfiguration(SubscriptionType subscriptionType)
 
         public string TopicName { get; set; }
         public string PublishEndpoint { get; set; }
+        public Dictionary<string, string> Tags { get; set; }
```

Comment:
Interface?

```suggestion
        public IDictionary<string, string> Tags { get; set; }
```

Reply:
The only thing with using the interface is that we'll need to use `Dictionary<string, string>` when creating the AWS request. I figured we could go with this as it's unlikely this will be used over the fluent API but happy to change if you think otherwise